lib/createResource.dart

```diff
@@ -208,6 +215,11 @@ class _CreateResourceState extends State<CreateResource> {
         'createdTime': FieldValue.serverTimestamp(),
       };
 
+      // Submit to admin submission if object not null
+      if(userSubmission != null) {
+        userSubmission?.submittedResource(resourceName, resourceType);
+      }
```

Comment:
I believe you don't need the `if` here -- `userSubmission?.submittedResource(...)` should skip calling the method if the userSubmission object is null.

Reply:
`?.` is part of a family of null-aware operations supported by Dart.